js/data.js

```diff
@@ -344,12 +388,23 @@ var data = {
         dozent: ["Prof. Dr. Andreas Polze", "Frank Feinbube", "Uwe Hentschel"],
         nameLV: "Eingebettete Betriebssysteme"
     },
+    eingebetteteBetriebssystemeWithITSE: {
+        kurz: "Embedded<br />Systems (mit ITSE)",
+        kennung: [IST, OSIS, SAMT, ITSE],
```

Comment:
Ok, same thing for ME310 and HCT then. I will fix the mistakes later this evening.